test/new-e2e/system-probe/config/platforms.json

```diff
@@ -4,7 +4,7 @@
         "ubuntu_16.04": "master/ubuntu-16-x86_64.qcow2.xz",
         "ubuntu_18.04": "master/ubuntu-18-x86_64.qcow2.xz",
         "ubuntu_20.04": "master/ubuntu-20-x86_64.qcow2.xz",
-        "ubuntu_22.04": "master/ubuntu-22-x86_64.qcow2.xz",
+        "ubuntu_22.04": "usama.saqib/ubuntu22-with-6.5/ubuntu-22-x86_64.qcow2.xz",
```

Comment:
We will prepare a separate ubuntu 22 w/ 6.5 image for you. Blocking this PR until that is done.

Reply:
Actually since this image didn't reproduce the issue in CI, is it worth adding at the moment?

Reply:
I have reverted the changes as it is not related to this PR and added tests to validate the change